components/compiler-families/arm-compilers-devel/SPECS/arm-compilers-devel.spec

```diff
@@ -0,0 +1,159 @@
+#----------------------------------------------------------------------------bh-
+# This RPM .spec file is part of the OpenHPC project.
+#
+# It may have been modified from the default version supplied by the underlying
+# release package (if available) in order to apply patches, perform customized
+# build/install configurations, and supply additional files to support
+# desired integration conventions.
+#
+#----------------------------------------------------------------------------eh-
+
+%include %{_sourcedir}/OHPC_macros
+%{!?PROJ_DELIM: %global PROJ_DELIM -ohpc}
+
+%define pname arm-compilers-devel
+
+Summary:   OpenHPC compatibility package for Arm HPC compiler
+Name:      %{pname}%{PROJ_DELIM}
+Version:   1.3.4
+Release:   1
+License:   Apache-2.0
+URL:       https://github.com/openhpc/ohpc
+Group:     %{PROJ_NAME}/compiler-families
+BuildArch: aarch64
+
+#!BuildIgnore: brp-check-suse
+#!BuildIgnore: post-build-checks
+
+Requires: lmod%{PROJ_DELIM}
+Provides: %{pname}%{PROJ_DELIM}
```

Comment:
This looks unnecessary. The is the same as Name: and usually provided by RPM automatically.

Reply:
I feel I had to do this for a specific reason, but can't recall the details. I'll pull that out for now.